src/test/java/com/github/piedpiper/node/stepfunctions/StepFunctionsExecuteTest.java

```diff
@@ -0,0 +1,157 @@
+		Mockito.when(AWSStepFunctionsClientBuilder.standard()).thenReturn(sfClientBuilder);
+		Mockito.when(sfClientBuilder.withRegion(Mockito.anyString())).thenReturn(sfClientBuilder);
+		Mockito.when(sfClientBuilder.withCredentials(Mockito.any())).thenReturn(sfClientBuilder);
+
+		//sfExecuteResult is a  StartExecutionResult object having fields with null value 
+		StartExecutionResult sfExecuteResult = new StartExecutionResult(); 
+		Mockito.when(sfClient.startExecution(Mockito.any())).thenReturn(sfExecuteResult);
+		Mockito.when(sfClientBuilder.build()).thenReturn(sfClient);
+		
+		
+		//mock StartExecutionRequest
+		PowerMockito.mock(StartExecutionRequest.class);
+		Mockito.doNothing().when(sfExecuteRequest).setName(Mockito.anyString());
+		Mockito.doNothing().when(sfExecuteRequest).setStateMachineArn(Mockito.anyString());
+		Mockito.doNothing().when(sfExecuteRequest).setInput(Mockito.anyString());
+		
+		
+	
+	}
+	
+	@Test
+	public void testSuccess() throws Exception {
+		
+		//sfExecuteResult is a StartExecutionResult object for testing 
+		StartExecutionResult sfExecuteResult = new StartExecutionResult();
+		sfExecuteResult.setExecutionArn("Sample_ARN");
+		sfExecuteResult.setStartDate(new Date(2020,5,15));
+		Mockito.when(sfClient.startExecution(Mockito.any())).thenReturn(sfExecuteResult);
+		
+		StepFunctionsExecute sfExecute = getStartFunctionExecuteNode();
+		Mockito.doReturn(sfExecuteRequest).when(sfExecute).getStartExecutionRequest();
+		
+		NodeInput input = new NodeInput();
+		input.setInput(JsonUtils.mapper.readTree(new FileInputStream(getFileName("SFExecuteSuccessGraph.json"))));
+		NodeOutput output = sfExecute.apply(input);
+		
+		Assert.assertNotNull(output.getOutput());
+		Assert.assertEquals("Sample_ARN",output.getOutput().get("executionArn").asText());
+		Assert.assertEquals("61550303400000",output.getOutput().get("startDate").asText());
+		Mockito.verify(sfClientBuilder).withRegion("sample_region");
+		Mockito.verify(sfExecuteRequest,Mockito.times(1)).setName("test1");
+		Mockito.verify(sfExecuteRequest).setStateMachineArn("sample_arn");
+		Mockito.verify(sfExecuteRequest).setInput("{}");
+
+	}
+	
+	@Test(expected = RuntimeException.class)
+	public void testRuntimeExceptio() throws Exception{
+		StepFunctionsExecute sfExecute = getStartFunctionExecuteNode();		
+		Mockito.doReturn(new RuntimeException()).when(sfExecute).getStartExecutionRequest();
+		NodeInput input = new NodeInput();
+		input.setInput(JsonUtils.mapper.readTree(new FileInputStream(getFileName("SFExecuteSuccessGraph.json"))));
+		sfExecute.apply(input);
+	}
+	
+	@Test
+	public void testParameterValidation() throws FileNotFoundException, IOException{
+		
+		StepFunctionsExecute sfExecute = getStartFunctionExecuteNode();
+		Mockito.doReturn(sfExecuteRequest).when(sfExecute).getStartExecutionRequest();
+		NodeInput input = new NodeInput();
+		input.setInput(JsonUtils.mapper.readTree(new FileInputStream(getFileName("emptyInput.json"))));
+		try{
```

Comment:
please use java standard formatter and ctlr+shift+F to format all files. This is doesn't seem right